lms/djangoapps/instructor/tests/test_tools.py

```diff
@@ -357,3 +359,51 @@ def msk_from_problem_urlname(course_id, urlname, block_type='problem'):
         urlname = urlname[:-4]
 
     return course_id.make_usage_key(block_type, urlname)
+
+
+@attr(shard=1)
+class TestStudentFromIdentifier(TestCase):
+    """
+    Test get_student_from_identifier()
+    """
+    @classmethod
+    def setUpClass(cls):
+        super(TestStudentFromIdentifier, cls).setUpClass()
+        cls.valid_student = UserFactory.create(username='baz@touchstone')
+        cls.duplicate_email = UserFactory.create(email='foo@touchstone.com')
+        cls.duplicate_user_name = UserFactory.create(username='foo@touchstone.com')
```

Comment:
Just a nit: I think it would be better to have names depicting conflict. May be something like `student_conflicting_email/username`. I am good with names `duplicate_email` and `duplicate_user_name` either, just a suggestion.